docs/source/ruletypes.rst

```diff
@@ -5,6 +5,80 @@ Examples of several types of rule configuration can be found in the example_rule
 
 .. _commonconfig:
 
+Rule Configuration Cheat Sheet
+==============================
+
+
++--------------------------------------------------------------------------------------------------------------------+
+|              FOR ALL RULES                                                                                         |
++==========================================+===========+=================================================+===========+
+| ``es_host`` (string)                     |           | ``buffer_time`` (time)                          | Optional  |
++------------------------------------------+           +-------------------------------------------------+           +
+| ``es_port`` (number)                     | Required  | ``query_delay`` (time)                          |           |
++------------------------------------------+-----------+-------------------------------------------------+           +
+| ``use_ssl`` (boolean, no default)        | Optional  | ``max_query_size`` (int, default 100k)          |           |
++------------------------------------------+           +-------------------------------------------------+-----------+
+| ``es_username`` (string, no default)     |           | ``filter`` (DSL filter, no default)             | Required  |
```

Comment:
Filter isn't actually required. (It was at one point, now it defaults to an empty list)